OPSEXP-1303 Document secrets present in the playbook (#305)
Alfresco/alfresco-ansible-deployment: docs/SECRETS.md

# Secrets

> This page describe which secrets and how they can be managed in our playbook.

Reviewer: ```suggestion
This page describe how secrets are securely managed in our playbook. It also serves as a comprehensive list of credentials in case you want to populated them manually. 
```

> This page describe how secrets are securely managed in our playbook. It also serves as a comprehensive list of credentials in case you want to populated them manually. 

Author: ```suggestion
This page describe how secrets are securely managed in our playbook. It also serves as a comprehensive list of credentials in case you want to populated them manually.
```

Links to the official documentation relevant for this subject:

* [Alfresco Authorization](https://docs.alfresco.com/content-services/latest/admin/security/)

## Secrets defined as variables

### activemq_password

The password used to access the activemq instance.

### repo_db_password

The password used to access the postgres database of Repository

### sync_db_password

The password used to access the postgres database of Sync

### reposearch_shared_secret

The secret used between Solr and Repository for communicating.

> ## Secrets not yet properly handled

Reviewer: ```suggestion
## Secrets not yet handled
```
